### PR TITLE
Add WritingGoal class

### PR DESCRIPTION
**Pull request by p-goulart**
- add WG<=>TT mapping to constants;
 - add test to make sure we can get the WGs correctly from the TTs;
 - add WG column to master CSV.

This PR adds writing goal info to the master CSV. We'll add this info to the summary (with rule counts) once @affemitkaraffe and I are done with #16 and #17.

(It's a simple change, but whoever wants to review it, please do. Joe won't be back until Tuesday.)

### CODE REVIEW

File: scripts/master_csv.py [hunk 1]
```diff
@@ -26,13 +27,15 @@ def __init__(self):
...
 # tone_tags
+# writing goals
 # is_goal_specific
 def __main__():
     cli = CLI()
     logger = logger_wrapper(cli.parser.prog, cli.args.verbosity)
     logger.debug(f"Starting script...\nInvoked with options: {cli.args}")
     out_path = path.join(cli.args.out_dir, 'all_rules.csv')
-    headers = ['id', 'subId', 'locale', 'source_repo', 'type', 'source_file', 'tone_tags', 'is_goal_specific']
+    headers = ['id', 'subId', 'locale', 'source_repo', 'type', 'source_file', 'tone_tags', 'writing_goal',
```
Comment (affemitkaraffe) [inline scripts/master_csv.py:37]: `tone_tags` and `writing_goals`
or
`tone_tag` and `writing_goal`
for consistency?

File: README.md [hunk 2]
```diff
@@ -100,11 +100,11 @@ ABSAGE_SUBST
 This script is meant to provide an all-encompassing view of **all** rules created by the lingu team in **one single file** – this means *all locales* are present in the same file. For example:
 
 ```csv
-row,id,locale,source_repo,type,source_file,tone_tags,is_goal_specific
-0,IN_SHANGHAI,en,os,grammar,grammar.xml,,false
-1,IN_SHANGHAI,en,os,grammar,grammar.xml,,false
-2,LOWERCASE_NAMES,fr,premium,grammar,grammar.xml,,false
-3,SOME_EXAMPLE,es,os,style,grammar.xml,professional,true
+row,id,subId,locale,source_repo,type,source_file,tone_tags,writing_goal,is_goal_specific
```
Comment (p-goulart) [inline README.md:103]:
```suggestion
row,id,subId,locale,source_repo,type,source_file,tone_tags,writing_goals,is_goal_specific
```

File: README.md [hunk 3]
```diff
@@ -118,6 +118,7 @@ row,id,locale,source_repo,type,source_file,tone_tags,is_goal_specific
 |type|'grammar', 'style', or 'unknown'|
 |source_file|the path to the file where this rule comes from|
 |tone_tags|comma-separated list of `tone_tags` applied to the the rule, including those inherited from rulegroups and categories|
+|writing_goal|comma-separate list of writing goals served by the tone tags in the previous column|
```
Comment (p-goulart) [inline README.md:121]:
```suggestion
|writing_goals|comma-separate list of writing goals served by the tone tags in the previous column|
```

File: scripts/master_csv.py (hunk 1 above)
Comment (p-goulart) [inline scripts/master_csv.py:37]:
```suggestion
    headers = ['id', 'subId', 'locale', 'source_repo', 'type', 'source_file', 'tone_tags', 'writing_goals',
```

**Review by affemitkaraffe — APPROVED** [review]
approved!

**Pull request merged**